### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by floresv**
### Summary

Add travis CI, configuration and status badge on readme file. Generated database.yml configuration for travis.

### Trello Card

https://trello.com/c/oYCrIo60/13-configurar-travisci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+before_install:
+  - "echo '--colour' > ~/.rspec"
+  - "echo 'gem: --no-document' > ~/.gemrc"
+  - gem update --system
+
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - psql -c "CREATE USER \"rails-bootstrap\" WITH PASSWORD 'rails-bootstrap';" -U postgres
+  - psql -c "DROP DATABASE IF EXISTS \"jf_rails_test\";" -U postgres
+  - psql -c "CREATE DATABASE \"jf_rails_test\" ENCODING 'utf8';" -U postgres
```
Comment (leamotta) [inline .travis.yml]: here you should have the user as specified in this card https://trello.com/c/MSPI1oDq/16-crear-aplicaci%C3%B3n-rails-wbooksapi
Same with the database name, it should be `wbooks_api_test`
Check if you created them correctly or it's just a copy/paste thing :D

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - bundle exec rake db:migrate
+
+script:
+  - bundle exec rubocop app spec -R --format simple
+
+cache:
+  bundler: true
```
Comment (leamotta) [inline .travis.yml]: Always remember to leave a newline at the end of each file

File: config/database.travis.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# PostgreSQL. Versions 9.1 and up are supported.
```
Comment (leamotta) [inline config/database.travis.yml]: I don't think you need this file 🤔 
Did you add it for any particular reason?

**Review by leamotta — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (floresv) [reply .travis.yml]: Ok

File: .travis.yml (hunk 2 above)
Comment (floresv) [reply .travis.yml]: Ok

File: config/database.travis.yml (hunk 3 above)
Comment (floresv) [reply config/database.travis.yml]: By default the travis.yml configuration file use it to replace the database.yml. I will remove it and set the user and password for test environment in the database.yml file.

**Review by leamotta — APPROVED** [review]

**Pull request merged**